lib/typescript/httpclient/BUILD

```diff
@@ -13,6 +15,51 @@ ts_web_library(
     ],
 )
 
+web_library(
+    name = "dist",
+    app_lib = ":httpclient",
+    entry = "typescript/httpclient/index.ts",
+    externals = {
+        "@types/node": "@types/node",
+        "camelcase-keys": "camelcase-keys",
+    },
+    output = {
+        "library": "@airyhq/httpClient",
```

Comment:
Npm modules are usually in `kebab-case`. I think we should go with that convention.